pkg/vtpm/init.sh

```diff
@@ -1,4 +1,4 @@
 #!/bin/sh
 
-#TBD, fill it up with the actual server launch
-tail -f /dev/null
+#Launch the VTPM server
+/usr/bin/vtpm_server > vtpm_server.log 2>&1
```

Comment:
Where does the logfile go? Would it make sense to put it in /persist/IMGx/log?
Can address that later.